server/src/test/java/org/elasticsearch/index/mapper/DocumentMapperTests.java

```diff
@@ -95,7 +95,7 @@ public void testMergeObjectAndNested() throws Exception {
                 IllegalArgumentException.class,
                 () -> MapperService.mergeMappings(objectMapper, nestedMapper.mapping(), reason)
             );
-            assertThat(e.getMessage(), containsString("can't merge a nested mapping [obj] with a non-nested mapping"));
+            assertThat(e.getMessage(), containsString("can't merge a non nested mapping [obj] with a nested mapping"));
```

Comment:
Minor grammatical nit
```suggestion
            assertThat(e.getMessage(), containsString("can't merge a non-nested mapping [obj] with a nested mapping"));
```